cluster/examples/kubernetes/ceph/monitoring/prometheus-ceph-v14-rules-external.yaml

```diff
@@ -1,9 +1,6 @@
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  labels:
```

Comment:
@aruniiird We would need to update the source for this generated file to remove these labels.

Reply:
Oh these are generated? Will take a closer look - thanks